src/main/webapp/app/course/course-scores/course-scores.component.ts

```diff
@@ -224,7 +227,7 @@ export class CourseScoresComponent implements OnInit, OnDestroy {
                         console.warn('found more than one result for student ' + student.user.login + ' and exercise ' + exercise.title);
                     }
 
-                    const pointsAchievedByStudentInExercise = (result.score! * relevantMaxPoints) / 100;
+                    const pointsAchievedByStudentInExercise = round((result.score! * relevantMaxPoints) / 100, 1);
                     student.overallPoints += pointsAchievedByStudentInExercise;
```

Comment:
I think we should round the overall points as late as possible and add the most accurate score we can archive here. Otherwise, small errors like the quarter points add up again.

Reply:
This decision was made with @krusche. If I understood it correctly, this is correct as we want to round on the individual exercise level.

Reply:
The problem with rounding late is as follows:
If we show the student 5 times that he has 1,1 points in an exercise while he actually has 1,05 then rounding the sum would be quite problematic, because suddenly the student would have 5,25 -> 5,3 points even if the actual sum of 5 times 1,1 points would lead to 5,5 points. Of course we could round to more decimal places but then the same issue can occur.
So I would think the best solution is to decide on the exercise level how to round and then take the sum of all rounded values. This will not lead to confusion. Also I don't think we should be too critical to small rounding errors on the exercise level. They will only occur on programming exercises in the future and won't affect manually corrected exercises, except reviewers give weiter points